docs/site/Inside-Loopback-Application.md

```diff
@@ -172,17 +172,17 @@ into repository classes.
 ## API Security
 
 > Security implementations in LoopBack can be created as separate
-> [Authentication strategies](Loopback-component-authentication.md) and the
-> `@authenticate` decorator can be used to define the authentication strategy of
-> a particular endpoint.
+> [Authentication strategies](https://loopback.io/doc/en/lb4/Loopback-component-authentication.html)
```

Comment:
It's better to use `.md` file for reference.

Reply:
reverted 👍